Documentation/project-docs/dogfooding.md

```diff
@@ -0,0 +1,77 @@
+# How to patch dotnet CLI with latest CoreFX
+
+This document provides the steps necessary to consume a nightly build of CoreFX
+and CoreCLR.
+
+Please note that these steps aren't necessary for official builds -- these steps
+are specficic to consuming nightlies and thus unsupported builds. Also note that
+these steps are likely to change as we're simplifying this experience. Make
+sure to consult this document often.
+
+## Update framework and CLI
+
+1. Add the CoreFX feed by editing
+   `C:\Users\XXX\AppData\Roaming\NuGet\NuGet.Config` to add this line:
+    ```
+    <add key="CoreFX" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" />
+    ```
+	or, if you have `nuget.exe` in your path, this will work too:
+    ```
+	nuget sources Add "foo" https://dotnet.myget.org/F/dotnet-core/api/v3/index.json
+    ```
+
+2. Pick a package version from
+   <https://dotnet.myget.org/feed/dotnet-core/package/nuget/Microsoft.NETCore.App>
+   feed. Presumably you want the latest version.
+
+3. Get CLI preview4+:
+    1. Install CLI preview 4 SDK from here:
+        - <https://github.com/dotnet/cli>
+        - <https://dotnetcli.blob.core.windows.net/dotnet/Sdk/rel-1.0.0/dotnet-dev-win-x64.latest.exe>
```

Comment:
Can nightlies be consumed on non-Windows platforms?  Assuming yes, it'd be helpful to include those differences as well, e.g. the paths and Urls to use on Linux and macOS.